org.eclim.jdt/java/org/eclim/plugin/jdt/command/complete/CodeCompleteCommand.java

```diff
@@ -185,6 +184,9 @@ protected CodeCompleteResult createCompletionResult(
     // of whether the user ever views it.
     /*return new CodeCompleteResult(
         kind, completion, menu, proposal.getAdditionalProposalInfo());*/
-    return new CodeCompleteResult(completion, menu, menu, type, offset);
+    CodeCompleteResult result = new CodeCompleteResult(completion, menu, menu, type, offset);
+    result.setRelevance(proposal.getRelevance());
```

Comment:
From my experience with eclim, it is hard to find an item in autocomplete if it is in the same package. With this option, autocomplete order becomes the same with Eclipse and it is easier to use IMHO.